packages/ds-healthcare-gov/src/components/Footer/Logo.jsx

```diff
@@ -35,6 +35,7 @@ function Logo(props) {
         'hc-c-footer__logo ds-u-display--inline-block',
         props.className
       )}
+      // eslint-disable-next-line react/no-danger
```

Comment:
[question] if we're comfortable with `dangerouslySetInnerHTML` used when necessary, can we change the linting rule to be a warning? Or does it make sense to keep it as an error and disable individual files like here

Reply:
Hmm, looks like apart from the child design system we only use it in the docs package. I think it would be worth going back to these and figuring out if we need it, because I do think we should try to avoid it. How do you feel about it?